apps/src/templates/JsDebugger.jsx

```diff
@@ -217,13 +227,20 @@ var JsDebugger = React.createClass({
         </PaneHeader>
         {this.props.debugButtons && <DebugButtons/>}
         {this.props.debugConsole && <DebugConsole debugButtons={this.props.debugButtons} debugWatch={showWatchPane}/>}
-        {showWatchPane && <DebugWatch debugButtons={this.props.debugButtons}/>}
+        <div style={{display: showWatchPane ? 'initial' : 'none'}}>
+          <ProtectedStatefulDiv>
+            <div id="watchersResizeBar"></div>
+          </ProtectedStatefulDiv>
+          </div>
```

Comment:
Indentation nit.  Also, is there a need for the extra layer of wrapping here?  I guess... does `showWatchPane` toggle the open/closed state of the watch area, or whether it exists at all?  If the latter, I think you can apply the style to the `ProtectedStatefulDiv` directly and be good to go (since it won't change unless the page reloads).